Bottom: 97158f672833c5b9077c3ea893891d5fca72ed34
Top:    3a0bce0208a7b209c89f07c620db8d2ef4afebc0
Author: Dan McMahill <dan@mcmahill.net>
Date:   2010-03-11 23:48:07 +0000

Revert "As silly as it seems to group any sort of scientific or EDA"

I've pushed really hard to sort this nonsense out, and this patch is
not in my opinion suitable for applcation. gEDA, gerbv and PCB all
deliberately omit the Education category.. this is not where we need
to fix this issue.

I strongly object to putting bogus categories into our apps.
If Debian want to patch these in, let them.

This reverts commit b402a144e19225f0648189bca6fbfad95d046047.



---

diff --git a/data/pcb.desktop.in b/data/pcb.desktop.in
index bc61016..61091b4 100644
--- a/data/pcb.desktop.in
+++ b/data/pcb.desktop.in
@@ -7,4 +7,4 @@ Type=Application
 Exec=pcb %f
 Icon=pcb
 MimeType=application/x-pcb-layout;application/x-pcb-footprint;
-Categories=Education;Science;Engineering;Electronics;
+Categories=Engineering;Electronics;
